src/components/DateInput.vue

```diff
@@ -43,6 +43,7 @@
             maxlength="4"
             v-on:keypress="isNumber($event)"/>
         <div class="date-picker-icon"
+            :data-cy="getCypressValue('DatePicker')"
```

Comment:
Maybe not to confuse this with the "DatePicker" component, can you rename this to `CalendarIcon` or `DatePickerIcon`?

Reply:
Fixed in 50d19245a59bbb4dfc9146c5396d641744a08f99